docs/robots/warthog/tutorials_warthog.mdx

```diff
@@ -4,9 +4,935 @@ sidebar_position: 4
+
+```
+rosrun rqt_robot_monitor rqt_robot_monitor
+```
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/warthog_images/rqt_robot_monitor.png"
+      width="600"
+    />
+    <figcaption>Robot Monitor</figcaption>
+  </figure>
+</center>
+
+### Driving Warthog {#driving-warthog}
+
+There are three ways to drive Warthog and each way will work on a physical Warthog robot as well as on a simulated Warthog.
+
+1.  Using the interactive remote controller in RViz. See [Simulating Warthog](#simulating-warthog).
+2.  Using the controller for teleoperation. See below.
+3.  Publishing ROS messages. See below.
+
+:::warning
+
+Warthog is capable of reaching high speeds.
+Careless driving can cause harm to the operator, bystanders, the robot, or other property.
+Always remain vigilant, ensure you have a clear line of sight to the robot, and operate the robot at safe speeds.
+We strongly recommend driving in normal (slow) mode first, and only enabling turbo in large, open areas that are free of people and obstacles.
+
+:::
+
+:::caution Pinch Points
+
+Refer to [Pinch Points](/docs/robots/warthog/user_manual_warthog#warthog-pinch-points) for
+pinch points on the Warthog that should be avoided during operation.
+
+:::
+
+#### Driving with Remote Controller
+
+Warthog uses a standard Futaba remote control radio transmitter. To operate the remote, first power it on by sliding
+the power switch to the ON position, as indicated in the image below.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/warthog_images/robot_warthog_10.png"
+      width="800"
+    />
+    <figcaption>Fubata Controller</figcaption>
+  </figure>
+</center>
+
+To drive the Warthog, the spring-loaded deadman switch in the upper-right corner must be held down. Releasing this switch will prevent the robot from moving.
+
+The lever on the left controls the robot's speed and the lever on the right controls the robot's rotation.
+
+#### Driving with ROS Messages
+
+You can manually publish `geometry_msgs/Twist` ROS messages to either the `/warthog_velocity_controller/cmd_vel` or the `/cmd_vel` ROS topics to drive Warthog.
+
+For example, in terminal, run:
+
+```
+rostopic pub /warthog_velocity_controller/cmd_vel geometry_msgs/Twist '{linear: {x: 0.5, y: 0.0, z: 0.0}, angular: {x: 0.0, y: 0.0, z: 0.0}}'
+```
+
+The command above makes Warthog drive forward momentarily at 0.5 m/s without any rotation.
+
+### Extending Warthog Startup
+
+Now that you've had Warthog for a while, you may be interested in how to extend it, perhaps add some more payloads, or augment the URDF.
+
+#### Startup Launch Context
+
+When ROS packages are grouped together in a directory and then built as one, the result is referred to as a workspace.
+Each workspace generates a `setup.bash` file which the user may source in order to correctly set up important environment variables such as `PATH`, `PYTHONPATH`, and `CMAKE_PREFIX_PATH`.
+
+The standard system-wide setup file is in `opt`:
```

Comment:
I suggest this be changed to '/opt' for people less familiar with Linux.

Reply:
Updated